common/thorhelper/thorsoapcall.cpp

```diff
@@ -2434,7 +2434,24 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
         unsigned retryInterval = 0;
 
         Url &url = master->urlArray.item(idx);
-        createHttpRequest(url, request);
+        StringBuffer spanName("SoapCall ");
+        url.getUrlString(spanName);
+
+        Owned<ISpan> clientSpan;
+        ISpan * activeSpan = master->logctx.queryActiveSpan();
+        clientSpan.setown(activeSpan->createClientSpan(spanName.str()));
```

Comment:
style: clearer to have the declaration of the variable at the same place it is initialised.  i.e.
```
Owned<ISpan> clientSpan(activeSpan->createClientSpan(spanName.str()));
```
or
```
Owned<ISpan> clientSpan = activeSpan->createClientSpan(spanName.str());
```

Reply:
> style: clearer to have the declaration of the variable at the same place it is initialised. i.e.
> 
> ```
> Owned<ISpan> clientSpan(activeSpan->createClientSpan(spanName.str()));
> ```
> 
> or
> 
> ```
> Owned<ISpan> clientSpan = activeSpan->createClientSpan(spanName.str());
> ```

Agreed. To be fair, I borrowed this from one of your commits:
https://github.com/hpcc-systems/HPCC-Platform/blame/cb3794d81c594a7be5c25cb7e9e99fa44ab0249c/esp/services/ws_ecl/ws_ecl_service.cpp#L1981C18-L1981C28

Looking through the history of that change does bring up a question about our ability to depend on activeSpan != nullptr